### PR TITLE
Move lib/last_traffic into app/lib

### PR DESCRIPTION
**Pull request by duduribeiro**
Extracted from #1396

In order to remove the lib folder from autoload, this commit moves the
lib/last_traffic into app/lib

Verification steps:

Running LastTraffic Worker should sent the traffic information to Segment.

### CODE REVIEW

File: app/lib/last_traffic.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
```
Comment (duduribeiro) [inline app/lib/last_traffic.rb]: please review this file entirely, I rewrite it to fix CodeClimate issues

**Comment by Martouta** [comment]
I would like to see verification steps in the description :smile: Because unfortunately, I cannot afford to figure out by myself for all these "move to app/lib" PRs :sob:

File: test/workers/last_traffic_worker_test.rb [hunk 2]
```diff
@@ -7,20 +7,20 @@ def setup
 
   def test_perform
     Timecop.freeze do
-      LastTraffic.expects(:sent_traffic_on).with(provider, Time.now).returns(Date.today)
+      LastTraffic.any_instance.expects(:sent_traffic_on).with(Time.now).returns(Date.today)
```
Comment (hallelujah) [inline test/workers/last_traffic_worker_test.rb]: I would avoid `any_instance` just stub it


```ruby
last_traffic = LastTraffic.new(provider)
LastTraffic.stubs(new: last_traffic)
last_traffic.expects(:sent_traffic_on).with(Time.now).returns(Date.today)
```

File: test/workers/last_traffic_worker_test.rb [hunk 3]
```diff
@@ -7,20 +7,20 @@ def setup
...
       LastTrafficWorker.new.perform(provider.id)
     end
   end
 
   def test_perform_no_traffic
-    LastTraffic.expects(:sent_traffic_on)
+    LastTraffic.any_instance.expects(:sent_traffic_on)
```
Comment (hallelujah) [inline test/workers/last_traffic_worker_test.rb]: Same we could avoid `any_instance`

File: test/workers/last_traffic_worker_test.rb [hunk 4]
```diff
@@ -7,20 +7,20 @@ def setup
...
     LastTrafficWorker.new.perform(provider.id)
   end
 
   def test_perform_yesterday
     date = Date.yesterday
     time = date.to_time
-    LastTraffic.expects(:sent_traffic_on).with(provider, time)
+    LastTraffic.any_instance.expects(:sent_traffic_on).with(time)
```
Comment (hallelujah) [inline test/workers/last_traffic_worker_test.rb]: Same avoid `any_instance`

**Pull request merged**